scenarioo-client/test/e2e/pages/stepPage.ts

```diff
@@ -238,18 +238,22 @@ class StepPage {
         return expect(element(by.id('comparison-tab')).isPresent()).toBeFalsy();
     }
 
-    async assertStepComparisonSideBySideViewIsActive() {
+    async assertStepComparisonSideBySideViewIsActive(otherScreenVisible: boolean) {
         await expect(element(by.id('sc-step-comparison-side-by-side-view-button')).getAttribute('class')).toContain('active');
         await expect(element(by.id('sc-step-comparison-current-screen-view-button')).getAttribute('class')).not.toContain('active');
-        await expect(element(by.id('sc-step-comparison-other-screen-view-button')).getAttribute('class')).not.toContain('active');
+        if (otherScreenVisible) {
+            await expect(element(by.id('sc-step-comparison-other-screen-view-button')).getAttribute('class')).not.toContain('active');
+        }
         await expect(element(by.id('sc-step-comparison-side-by-side-view')).isPresent()).toBeTruthy();
         return expect(element(by.id('sc-step-comparison-single-page-view')).isPresent()).toBeFalsy();
     }
 
-    async assertStepComparisonCurrentScreenViewIsActive() {
+    async assertStepComparisonCurrentScreenViewIsActive(otherScreenVisible: boolean) {
```

Comment:
Imho this should be a separate method/function not a flag. The usage of the method is strange to me.

Reply:
Added separate function.